compiler/src/main/java/com/squareup/anvil/compiler/Utils.kt

```diff
@@ -276,3 +276,6 @@ internal fun FqName.safePackageString(
     val suffix = if (dotSuffix) "." else ""
     "$prefix$this$suffix"
   }
+
+internal fun String.capitalizeSupport(): String = replaceFirstChar(Char::uppercaseChar)
```

Comment:
Nit: Why bother with the support moniker? Just leave it as String.capitalize or String.decapitalize.

Reply:
You're right. When I did this here a while ago I was afraid of the conflicting overload with the standard library. But I've seen in our codebase that it isn't an issue. Will change.